firefox-ios-inactive-tabs.toml

```diff
@@ -0,0 +1,99 @@
+[experiment]
+
+segments = ["has_10_plus_tabs"]
+
+[segments.has_10_plus_tabs]
+select_expression = 'MAX(max_tabs) >= 10' # client had 10+ tabs any day in the two weeks prior to enrollments
+data_source = "daily_max_tabs"
+
+[metrics]
+overall = ["week_1_retention", "week_3_retention"]
+
+[metrics.week_1_retention]
+select_expression = "LOGICAL_OR(retention.day_27.active_in_week_1)"
+data_source = "retention"
+friendly_name = "Retention Week 1"
+description = "Percentage of clients that are active on week 1"
+[metrics.week_1_retention.statistics.binomial]
+
+[metrics.week_3_retention]
+select_expression = "LOGICAL_OR(retention.day_27.active_in_week_3)"
+data_source = "retention"
+friendly_name = "Retention Week 3"
+description = "Percentage of clients that are active on week 3"
+[metrics.week_3_retention.statistics.binomial]
+
+## Data Sources
+[segments.data_sources.daily_max_tabs]
+from_expression = """(
+    SELECT
+        date as submission_date,
+        client_id,
+        MAX(average_tabs) AS max_tabs
+    FROM (
+        SELECT
+            date,
+            seq,
+            c.client_id,
+            MAX(tabs_cumulative_count) AS cumulative_tabs,
+            COUNT(1) AS inactive,
```

Comment:
I'm guessing these are just here for documentation purposes?

Reply:
ah, yes, I'll remove these two lines since they are not used in the query.